src/assets/fonts.ts

```diff
@@ -0,0 +1,9 @@
+import '@fontsource/archivo/400.css';
```

Comment:
Should only include latin:

```ts
import '@fontsource/archivo/latin-400.css';
import '@fontsource/archivo/latin-600.css';
```

As it currently looks like this:
```css
@font-face {
    font-family: Archivo;
    font-style: normal;
    font-display: swap;
    font-weight: 400;
    src: url(/_astro/archivo-vietnamese-400-normal.CQ9LHvVU.woff2) format("woff2"),url(/_astro/archivo-vietnamese-400-normal.zAKXS6v3.woff) format("woff");
    unicode-range: U+0102-0103,U+0110-0111,U+0128-0129,U+0168-0169,U+01A0-01A1,U+01AF-01B0,U+0300-0301,U+0303-0304,U+0308-0309,U+0323,U+0329,U+1EA0-1EF9,U+20AB
}

@font-face {
    font-family: Archivo;
    font-style: normal;
    font-display: swap;
    font-weight: 400;
    src: url(/_astro/archivo-latin-ext-400-normal.CrL9qeB-.woff2) format("woff2"),url(/_astro/archivo-latin-ext-400-normal.DaZLa428.woff) format("woff");
    unicode-range: U+0100-02AF,U+0304,U+0308,U+0329,U+1E00-1E9F,U+1EF2-1EFF,U+2020,U+20A0-20AB,U+20AD-20C0,U+2113,U+2C60-2C7F,U+A720-A7FF
}

/* etc */
```

Reply:
Astro/Vite bundles the font families somewhere in a separate CSS file, so the browser still needs to fetch this CSS file before text can be rendered. Would be better if this could be inlined. Attempt on separate branch: https://github.com/voorhoede/head-start/commit/5a9cad9794dddc9205795cda6c70243befee81d6